rdt-open-reader/src/main/res/layout/activity_login.xml

```diff
@@ -36,12 +36,12 @@
             <TextView
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:gravity="center"
                 android:text="@string/login_app_name"
                 android:textAllCaps="false"
+                android:textColor="@color/black"
                 android:textSize="30sp"
-                android:gravity="center"
-                android:textStyle="bold"
```

Comment:
What do the changes here address?

Also, the hardcoded credentials should be removed.